src/components/30-organisms/table-sortable/ui.test.js

```diff
@@ -153,6 +153,52 @@ test('should sort numbers', async t => {
     .eql('<!----><span>55</span><!---->');
 });
 
+test('should sort dates', async t => {
+  const $el = await Selector('axa-table-sortable');
+  await t.expect($el.exists).ok();
+  const $columnOne = await Selector(() => {
+    const sRoot = document.querySelector('axa-table-sortable').shadowRoot;
+    return sRoot.querySelectorAll('th')[3];
+  });
+
+  const $columnFourFirstRow = await Selector(() => {
+    const sRoot = document.querySelector('axa-table-sortable').shadowRoot;
+    const firstRow = sRoot.querySelectorAll('tbody tr')[0];
+    return firstRow.querySelectorAll('td')[3];
+  }).addCustomDOMProperties({
+    innerHTML: el => el.innerHTML,
+  });
+
+  const $columnFourLastRow = await Selector(() => {
+    const sRoot = document.querySelector('axa-table-sortable').shadowRoot;
+    const firstRow = sRoot.querySelectorAll('tbody tr')[5];
+    return firstRow.querySelectorAll('td')[3];
+  }).addCustomDOMProperties({
+    innerHTML: el => el.innerHTML,
+  });
+
+  await t
+    .click($columnOne)
+    .expect($columnOne.getAttribute('aria-sort'))
+    .eql('ascending');
+  await t
+    .expect(await $columnFourFirstRow.innerHTML)
+    .eql('<!----><span>02.05.2013</span><!---->');
```

Comment:
This is dangerous. The comment strings are an artifact of lit-html's internal representation, and may change at any moment at their sole discretion. Can't you use `.textContent` here (and everywhere else) and compare against the serialized date alone?

Reply:
done

Reply:
I'll implement both cases, because it was requested from a team member to also test if the elements are rendered.
`await t
    .expect(await $columnFourFirstRow.innerHTML)
    .contains('<span>02.05.2013</span>');`